content/authors/aldrin.montana/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+# Display name
+title: Aldrin Montana
+
+# Username (this should match the folder name)
+authors:
+- aldrin.montana
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: PhD Candidate
+
+# Organizations/Affiliations
+organizations: UC Santa Cruz
```

Comment:
Build failed because `organizations:` requires a list of `name:`/`url:` pairs. For example:
```
organizations:
- name: Center for Computing Research (CCR)
  url: https://www.sandia.gov/ccr/
- name: Sandia National Laboratories
  url: https://www.sandia.gov/
```

Reply:
thanks! I couldn't decipher the error message